### PR TITLE
added section on goal spec clarity and goal improvement

### PR DESCRIPTION
**Pull request by jaredatron**

### CODE REVIEW

File: Learning_Guide/Goals.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Goals
+
+- [How do I get clarity on specs that are unclear?](/Learning_Guide/Goals/Clarity_on_specs.md).
+- [How do I submitting a PR against a Goal?](/Learning_Guide/Goals/Improvement.md).
```
Comment (AbrahamFergie) [inline Learning_Guide/Goals.md]: just a grammar check of "submitting" to be "submit"

**Review by AbrahamFergie — COMMENTED** [review]
Looks like it hits all the necessary areas, maybe have one fyi in the improvement file to states that they might not see the change anywhere from a couple days up to the following monday.

File: Learning_Guide/Goals/Improvement.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+0. Go [here](https://github.com/GuildCrafts/web-development-js/tree/master/_goals)
+0. Find your goal and click on it
+0. Click the Pencil
+0. Edit the Markdown file. _either make changes or ask for the changes you want_
+0. **IMPORTANT** make sure you select `Create a new branch for this commit and start a pull request`
+0. Click `Propose file change`
+
+
+_Note: It can take as much as a few weeks for your change requests to be addressed. Please be patient_
```
Comment (tannerwelsh) [inline Learning_Guide/Goals/Improvement.md:14]: Really? That's unfortunate if that's the impression. I've tried to address PRs within 24 hours, don't think I've missed any so far.

Issues, on the other hand, are stacking up for a long time into the future.

Comment (deadlyicon) [inline Learning_Guide/Goals/Improvement.md]: OMG sorry I did not mean to offend you. I honestly have no idea how long these normally take to get integrated. I just wanted learner expectations to be low. What would you prefer this to say?

**Pull request merged**